EnglishFilter/sections/foreign.txt

```diff
@@ -33,6 +33,9 @@ video.gjirafa.com#%#AG_defineProperty('originVideo.requestA.Id', { value: "0" })
 !------------------------------ Arabian ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78171
+hdlink.online#$#.adBanner { height:2px; !important width:2px; !important visibility:hidden; !important }
+@@||hdlink.online/css/ads.css^$important,domain=hdlink.online
```

Comment:
1. You have mistakes here `hdlink.online#$#.adBanner { height:2px; !important width:2px; !important visibility:hidden; !important }`.
2. `@@||hdlink.online/css/ads.css^$important,domain=hdlink.online` this rule can be limited for the cb platforms. `^$important,domain=hdlink.online` this is unnecessary part.